src/parse/proto/extensions/simple_extension_declaration/simple_extension_declaration.rs

```diff
@@ -0,0 +1,41 @@
+use crate::{
+    parse::{Context, Parse},
+    proto,
+};
+use thiserror::Error;
+
+use super::mapping_type::MappingType;
+
+// SimpleExtensionDeclaration
+#[derive(Clone, Debug, PartialEq)]
+pub struct SimpleExtensionDeclaration {
+    pub mapping_type: Option<MappingType>,
```

Comment:
Following the validator (https://github.com/substrait-io/substrait-validator/blob/3294676f7efbf166c87a3676f61897c0bcbe671c/rs/src/parse/extensions/simple/mod.rs#L393) this field is required, and should not be `pub` to prevent invalidation through direct mutation.
```suggestion
    mapping_type: MappingType,
```